docker/runtime/Dockerfile.pytorch

```diff
@@ -1,5 +1,5 @@
-ARG BASEIMAGE=nvidia/cuda:10.0-cudnn7-devel-ubuntu18.04
-FROM ${BASEIMAGE}
+ARG RUNTIME_BASEIMAGE=nvidia/cuda:11.0-cudnn8-devel-ubuntu18.04
```

Comment:
I prefer to remove the `RUNTIME` prefix, because
1. this `Dockerfile` is under the `runtime` folder
2. to be consistent with TF runtime Dockerfile.